test/abel.lean

```diff
@@ -98,15 +98,8 @@ error: abel_nf made no progress
 example [AddCommGroup α] (x y z : α) (_w : x = y + z) : False := by
   abel_nf at *
 
-/--
-error: no goals to be solved
--/
--- This error message is confusing: it is saying that it closed the main goal,
--- and so then had nothing to do on the hypotheses.
--- The user has to guess that they should remove the `at *`.
-#guard_msgs in
 example [AddCommGroup α] (x y z : α) (_w : x = y + z) : x - x = 0 := by
-  abel_nf at *
```

Comment:
As commented in https://github.com/leanprover-community/mathlib4/pull/9011/files#r1426012984, this looks a bit odd; I think this test was intended to test the error message for `at *`, not to succeed.

Reply:
The context here is https://github.com/leanprover/lean4/pull/2917.

The test was previously checking for an error message which we decided shouldn't be produced at all!

Reply:
I've changed the tactic back to `abel_nf at *`, and added a comment explaining the history.